pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -488,32 +500,6 @@ func (r *Reconciler) validateApiDependencies(ctx context.Context, th *v1alpha1.T
 			return err
 		}
 	}
-
-	_, err = r.getConfigMap(ctx, api, namespace, apiConfigMapKeys)
```

Comment:
will upgrade work fine here? will the old configmap will be removed.

Reply:
yes and  it's removing